keytransform/interface.go

```diff
@@ -16,6 +16,8 @@ type KeyTransform interface {
 type Datastore interface {
 	ds.Shim
 	KeyTransform
+
+	Batch() (ds.Batch, error)
```

Comment:
> change the entire interface name? or have a new interface for batching?

i just meant maybe embed the interface instead of the method name, but embedding the interface will probably clash with `ds.Shim` so nvm.